src/main/java/core/basesyntax/model/figure/Circle.java

```diff
@@ -1,5 +1,8 @@
 package core.basesyntax.model.figure;
 
+import javax.persistence.Entity;
+
+@Entity
 public class Circle extends Figure {
```

Comment:
what about Triangle? I can't see that class changed

Reply:
forget about it... fixed